x/namecoin/handler_test.go

```diff
@@ -241,6 +241,10 @@ func TestSetNameHandler(t *testing.T) {
 		// cannot create conflict - only checked deliver?
 		8: {perm, []orm.Object{newUser, dupUser}, msg,
 			noErr, orm.IsUniqueConstraintErr, nil, nil},
+		// cannot change - no such a wallet (should should up by addr2 not addr)
```

Comment:
this does check the error (good), but can you ensure the content of the rendered error (casting to string) only contains ascii? that would test the fix you made.
that can go in errors_test.go or other such place, not really a handler test (although I like the test you added as well)